python/whylogs/core/view/dataset_profile_view.py

```diff
@@ -77,6 +77,12 @@ def dataset_timestamp(self, date: datetime) -> "DatasetProfileView":
         self._dataset_timestamp = date
         return self
 
+    def set_dataset_timestamp(self, dataset_timestamp: datetime) -> None:
+        if dataset_timestamp.tzinfo is None:
+            logger.warning("No timezone set in the datetime_timestamp object. Default to local timezone")
```

Comment:
can we call the ensure timezone function so that there is an error raised if there is no timezone?